string behavior
geometry_msgs/Point target_eff_position
float64 time_to_execute
